migrations/Version202205232110572348_generis.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\generis\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\generis\model\data\Ontology;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use core_kernel_persistence_smoothsql_SmoothModel as SmoothModel;
+
+final class Version202205232110572348_generis extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Fix ontology models.';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $serviceLocator = $this->getServiceLocator();
+
+        /** @var SmoothModel $ontology */
+        $ontology = $this->getServiceLocator()->get(Ontology::SERVICE_ID);
+
+        $this->fixModels($ontology, SmoothModel::OPTION_READABLE_MODELS);
+        $this->fixModels($ontology, SmoothModel::OPTION_WRITEABLE_MODELS);
+        $this->fixModel($ontology, SmoothModel::OPTION_NEW_TRIPLE_MODEL);
```

Comment:
I would like that this migration makes the config to match https://github.com/oat-sa/generis/blob/master/config/default/ontology.conf.php#L8-L14

Reply:
If we change the migration to write a hardcoded list (as the one from default/generis.conf.php), I think we would lose any additional values that may be set by extensions (for example, https://github.com/oat-sa/extension-tao-workspace/blob/master/scripts/install/SetupWrapper.php)

Reply:
There is also the possibility to keep the values already stored in the config (as this migration does) but add also the default values if not present (maybe something like `array_unique(array_merge($defaults, $current_values))`, but I suppose it may be cases of users removing the default writable models on purpose.

I would keep this migration as it is.

Reply:
I cannot do this.
We have environments, which contains other values, and I don't want to reset them to default values.
@augustas 

![image](https://user-images.githubusercontent.com/59471572/169994573-4562025f-9ce4-4f86-976a-b29ea95fba89.png)

Reply:
Why you don't want to reset them to default? What is the purpose of those extra values?

Reply:
That's the case - I do not know why we need these values
Resetting them to default might break something

Reply:
If we will not clean up the code and config then in time it will be harder and harder to maintain it, code will rot and product will fail. So cleanup whenever possible, boy scout rule :) https://github.com/oat-sa/generis/pull/737 removes model table, it was already [a deprecated feature](https://github.com/oat-sa/generis/pull/731#pullrequestreview-359879262). So please, keep it only if you find code that actually needs it

Reply:
@augustas there might be the case some customer persisted the `model_id` in the database using those additional values, so it is a risky action IMO, to just remove them.

The lesson from this is that we should not create config and customization for unnecessary things, but now that is done, we can only keep it backward compatible IMO.

Reply:
Due to this requested changes, I cannot merge this PR 😢